ecdsa/party/implementation.go

```diff
@@ -295,6 +301,8 @@ func (signer *singleSigner) attemptToCacheIfUnauthorizedToSign(message tss.Parse
 	return
 }
 
+var ErrNotInSigningCommittee = errors.New("self not in signing committee")
+
 func (p *Impl) authoriseSignerToSign(digest Digest, signer *singleSigner) error {
```

Comment:
They are all authorised. Consider renaming this function since it's more about should this guardian sign.

Reply:
I changed it to `tryStartSigning` and added comments indicating what might happen in the function.